Tesla/Tesla/Helper/View/CustomStepper.swift

```diff
@@ -0,0 +1,30 @@
+// CustomStepper.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомный счетчик
+struct CustomStepper: View {
+    @Binding var isStarted: Bool
+
+    @Binding var value: Int
+
+    var body: some View {
+        HStack {
+            Button {
+                value -= 1
```

Comment:
```suggestion
                value -= 1
```
1 кто это и откуда? Что за маг число?